server_src/runners/history.js

```diff
@@ -346,45 +326,37 @@ class HistoryCollection {
       return null;
     }
   }
+
+  /**
+   * Grabs the timelapse by downloading it from OctoPrint's API
+   * @param fileName
+   * @param url
+   * @param id
+   * @param printer
+   * @param serverSettings
+   * @returns {Promise<string>}
+   */
   static async grabTimeLapse(fileName, url, id, printer, serverSettings) {
-    const download = async (url, path, callback) => {
```

Comment:
Moved to its own util. Also note I spotted copied but slightly different code in this file. Something for later?

Reply:
I'll be dealing with majority of this on the history fixer up branch so aye